pkg/transport/queuestorage/file_test.go

```diff
@@ -186,14 +185,17 @@ func TestFileWriterAutoRotation(t *testing.T) {
 		param := param
 		param.MaxFileAge = 1 * time.Second
 		checkRotation(t, param, func() {
-			// +10ms to ensure rotation completes from timer expiry
-			time.Sleep(param.MaxFileAge + (10 * time.Millisecond))
+			// +1s to ensure rotation completes from timer expiry
+			time.Sleep(param.MaxFileAge + (1 * time.Second))
```

Comment:
Is there a reason for this increase?

Reply:
If the machine is shared/busy, the filing writing, etc. can take longer than expected. 500ms, seemed a good value when testing with locally with `test -exec 'cpulimit ...'`, but the test already takes many seconds so I just increased it to a full second.